src/Database/DBStructure.php

```diff
@@ -128,6 +128,11 @@ private static function tableStructure($table) {
 		}
 		if (DBM::is_result($structures)) {
 			foreach ($structures AS $field) {
+				// Replace the default size values so that we don't have to define them
+				$search = ['tinyint(1)', 'tinyint(4)', 'smallint(6)', 'mediumint(9)', 'bigint(20)', 'int(11)'];
```

Comment:
What if the current field is `int(10) unsigned` like it is in the `challenge` table right now? Will it be changed automatically even if it isn't in the replacement list?

Reply:
The replacement list is only for the standard length. The ```int(10) unsigned``` will be converted to ```int``` anyway. Have a look in the definition of the challenge table in my PR.

Reply:
Ok, perfect.